WORKSPACE

```diff
@@ -2,7 +2,7 @@ workspace(name = "com_github_istio_mixer")
 
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "2c6fa767100b95799043451aa3ee221d9a9bbb55",
+    commit = "7828452850597b52b49ec603b23f8ad2bcb22aed",
```

Comment:
Do we want to pull a random CL from their HEAD? They tag releases, the most recent release in that repo is 0.4.0 which is commit 76c63b5cd0d47c1f2b47ab4953db96c574af1c1d (16 days old).